src/Common/tests/System/Net/HttpTestServers.cs

```diff
@@ -7,7 +7,7 @@ namespace System.Net.Tests
 {
     internal class HttpTestServers
     {
-        public const string Host = "corefx-net.azurewebsites.net";
+        public const string Host = "corefx-net.cloudapp.net";
```

Comment:
Good point.  I will file a TODO to fix this.  But renaming the public `Host` to `HttpHost` will impact more files to change in the repo.